packages/web/src/common/store/remote-config/sagas.ts

```diff
@@ -1,27 +1,29 @@
-import { RemoteConfigInstance } from '@audius/common'
 import { eventChannel, END } from 'redux-saga'
-import { put, take } from 'redux-saga/effects'
+import { put, take } from 'typed-redux-saga'
+
+import { getContext } from '../effects'
 
 import { setDidLoad } from './slice'
 
 const CLIENT_READY_EVENT = 'CLIENT_READY'
 
-const remoteConfigSagas = (remoteConfigInstance: RemoteConfigInstance) => {
-  function* watchRemoteConfigLoad() {
-    // Emit event when provider is ready
-    const chan = eventChannel((emitter) => {
-      remoteConfigInstance.onClientReady(() => {
-        emitter(CLIENT_READY_EVENT)
-        emitter(END)
-      })
-      return () => {}
+function* watchRemoteConfigLoad() {
+  const remoteConfigInstance = yield* getContext('remoteConfigInstance')
```

Comment:
It might be nice to do some context helper functions like `getRemoteConfigInstance()` or something

Reply:
yeah very down to figure out a less verbose pattern here